api/metrics/src/main/java/io/opentelemetry/api/metrics/common/ArrayBackedLabels.java

```diff
@@ -22,7 +22,7 @@ static Labels empty() {
   }
 
   static Labels sortAndFilterToLabels(Object... data) {
-    return new ArrayBackedLabels(sortAndFilter(data, /* filterNullValues= */ false));
+    return new ArrayBackedLabels(sortAndFilter(data, /* filterNullValues= */ true));
```

Comment:
I think we might be filtering null values everywhere now...so maybe we can just set true in the method itself?

Reply:
Followup :) don't want to change everything in one shot :)